content/en/cloud_cost_management/container_cost_allocation.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Container Cost Allocation
+kind: documentation
+disable_toc: false
+private: true
+further_reading:
+- link: "https://docs.datadoghq.com/cloud_cost_management/?tab=aws#overview"
+  tag: "Documentation"
+  text: "Cloud Cost Management"
+---
+
+{{< jqmath-vanilla >}}
+
+## Overview
+
+Datadog Cloud Cost Management (CCM) automatically allocates EC2 compute cost in Kubernetes and ECS clusters to individual pods and tasks running in those clusters. Use cost metrics enriched with tags from pods, nodes, containers, and tasks to visualize container workload cost in the context of your entire cloud bill.
+
+{{< img src="cloud_cost/container_cost_allocation/cost_allocation_table.png" alt="Your image description" style="width:100%;" >}}
+
+## Prerequisites
+
+1. Set up and configure [AWS Cloud Cost integration][1].
+1. At least one of the following must be running:
+    - [**Datadog Kubernetes agent**][2] in a Kubernetes environment using EC2 instances. 
+    - **Datadog container monitoring** in containers running in ECS tasks. The container agent must be configured to use [orchestrator tag cardinality][3].
+
+## Cost metrics
+
+When the prerequisites are met, new AWS cost metrics automatically appear.
+
+| AWS Cost Metric                    | Description    |
+| ---                                | ----------- | 
+| `aws.cost.amortized.mem.allocated`   | EC2 costs allocated by memory requested by a pod or ECS task. <br> *Based on `aws.cost.amortized`* | 
+| `aws.cost.net.amortized.mem.allocated` | Net EC2 costs allocated by memory requested by a pod or ECS task <br> *Based on `aws.cost.net.amortized`, if available* |
+| `aws.cost.amortized.cpu.allocated` | EC2 costs allocated by CPU requested by a pod or ECS task <br> *Based on `aws.cost.amortized`* |
+| `aws.cost.net.amortized.cpu.allocated` | Net EC2 costs allocated by CPU requested by a pod or ECS task <br> *Based on `aws.cost.net.amortized`* |
```

Comment:
Should we add `if available*` here as well since we have it for `aws.cost.net.amortized.mem.allocated`?

Reply:
```suggestion
| `aws.cost.net.amortized.cpu.allocated` | Net EC2 costs allocated by CPU requested by a pod or ECS task <br> *Based on `aws.cost.net.amortized`, if available* |
```